src/HealthGPS/energy_balance_model.cpp

```diff
@@ -51,7 +51,7 @@ void EnergyBalanceModel::update_risk_factors(RuntimeContext &context) {
             current_risk_factors.at(age_key) = model_age;
         }
 
-        double energy_intake = 0.0;
+        [[maybe_unused]] double energy_intake = 0.0;
```

Comment:
I assume the new linters pick this kind of stuff out automatically now, right?

Reply:
I didn't add new linters -- just new compilers -- but, yes, clang noticed that this variable was unused even though gcc didn't.